trailblazer/mip/trending.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+""" Get MIP data for trending."""
+
+from trailblazer.mip import files
+
+
+def parse_mip_analysis(mip_config_raw: dict, qcmetrics_raw: dict, sampleinfo_raw: dict) -> dict:
+    """Parse the output analysis files from MIP for adding info
+    to trend database
+
+    Args:
+        mip_config_raw (dict): raw YAML input from MIP analysis config file
+        qcmetrics_raw (dict): raw YAML input from MIP analysis qc metric file
+        sampleinfo_raw (dict): raw YAML input from MIP analysis qc sample info file
+    Returns:
+        dict: parsed data
+    """
+
+    ### Define output dict
```

Comment:
What's up with these triple `#`?

Reply:
It's just a way of commenting what the next block of code is handling. If you have another system in place in trailblazer feel free to modify accordingly.

Reply:
Python has a style guide which we are following: https://www.python.org/dev/peps/pep-0008/
e.g. Pycharm automatically highlights any deviations from pep-8.